weekly-updates/this-week-in-solid-2019-09-12.md

```diff
@@ -61,7 +61,7 @@ These releases do not contain major changes; rather, marking them as v1 merely r
 
 #### Introducing user types Developer and Power User
 
-The data browser team at inrupt is dedicated to make the Solid data browser the best that it can be. As part of this work, we wanted to improve the onboarding experience for new users, and we identified the number of views (aka panes) as one of the challenges for new users. Some of these views are also difficult to use, and we concluded that it might be prudent to limit some of the views shown.
+The data browser team at inrupt is dedicated to improve the Solid data browser. As part of this work, we wanted to improve the on-boarding experience for new users, and we identified the number of views (aka panes) as one of the challenges for new users. Some of these views are also difficult to use, and we concluded that it might be prudent to limit some of the views shown.
```

Comment:
`dedicated to improve` -> `dedicated to improving`

Reply:
Always the keen eye, thanks @TallTed :)

Protip: if you click the Suggestion button (see below), you can suggest a new line that can be directly incorporated with the click of a button.

![image](https://user-images.githubusercontent.com/4251/65018644-df9cea00-d929-11e9-8b88-2a18cd875d4c.png)

Example:

```suggestion
The data browser team at inrupt is dedicated to improving the Solid data browser. As part of this work, we wanted to improve the on-boarding experience for new users, and we identified the number of views (aka panes) as one of the challenges for new users. Some of these views are also difficult to use, and we concluded that it might be prudent to limit some of the views shown.
```

Reply:
The latest commit addresses this - sorry for not fixing the form in the previous commit.

Reply:
Thanks, @Vinnl, that button is far from obvious in the UI!